src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Speech.csproj">
+      <SupportedFramework>netcoreapp3.0;net45;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
I would have thought this just targets `netstandard2.0`, akin to registry and others?

Reply:
Here's a PR that does this: https://github.com/danmosemsft/runtime/pull/8